README.md

```diff
@@ -40,4 +60,42 @@ $ cd build.release
 $ cmake .. -DCMAKE_TOOLCHAIN_FILE=../submodules/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Release -DUSE_CPPRESTSDK=true
 $ cmake --build . --config Release
 ```
-Output will be in `build.release/bin/`
\ No newline at end of file
+Output will be in `build.release/bin/`
+
+## Example usage
+
+```cpp
+#include "signalrclient/hub_connection.h"
+#include "signalrclient/hub_connection_builder.h"
+#include "signalrclient/signalr_value.h"
```

Comment:
```suggestion
#include <iostream>
#include "signalrclient/hub_connection.h"
#include "signalrclient/hub_connection_builder.h"
#include "signalrclient/signalr_value.h"
#include <future>
```

iostream currently has to be first or else there's the issue including log_writer.h I mentioned in the other comment.

Reply:
Should be fixed by adding `<string>` to log_writer